server/profile.go

```diff
@@ -15,12 +15,80 @@
 package main
 
 import (
-    "net/http" 
+    "net/http"
+    "cloud.google.com/go/datastore"
+    "github.com/gorilla/mux"
+    "context"
+    "fmt"
+    "google.golang.org/api/iterator"
+    "log"
+    "encoding/json"
 )
 
+func CheckDatastore(w http.ResponseWriter, r *http.Request) {
+    ctx := context.Background()
+    client, err := datastore.NewClient(ctx, "lauraod-step-2020")
+    if err != nil {
+        w.WriteHeader(http.StatusOK)
+        w.Write([]byte(`first save error`))
```

Comment:
You don't need raw strings here and below. Usually in Go you just use normal strings.